R/rows-dm.R

```diff
@@ -256,6 +256,8 @@ do_rows_insert <- function(x, y, by = NULL, ..., autoinc_col = NULL) {
 do_rows_append <- function(x, y, by = NULL, ..., in_place = FALSE, autoinc_col = NULL) {
   if (is.null(autoinc_col)) {
     return(rows_append(x, y, ..., in_place = in_place))
+  } else if (inherits(x, "data.frame")) {
+    return(rows_append_ai_local(x, y, ..., autoinc_col))
```

Comment:
Do we need `rows_append_ai_remote()` too?

Each function should operate on the same level of abstraction. We could also get rid of the `return` statements then.